### ISSUE TITLE
Fixes for mobile platforms

### ISSUE THREAD

**Issue by irmod** [vrtmrz/obsidian-tagfolder#67]
Hello,

I love your wonderful TagFolder plugin!..

Could you fix the TagFolder panel layout for the mobile platform (Android)?

## 1. Transfer a row of buttons down

On Android, the panel buttons are placed at the bottom. This position is convenient for finger control.

Now TagFolder v.0.15.19 shows a row of buttons at the top...

![TagFolder-01519_1](https://user-images.githubusercontent.com/125995328/220713157-9481f483-1436-49be-bb97-f47af3b5c479.png)

I would like that:

![TagFolder-01519_2](https://user-images.githubusercontent.com/125995328/220713433-b3ae1698-6e72-4719-87a4-364f81bdd4e1.png)

## 2. Transfer a search field out of the scrolling area

Now the search field is opened inside the scrolling area

![TagFolder-01519_3](https://user-images.githubusercontent.com/125995328/220713599-3f4f6481-fd08-46dd-a690-f6d54ec90420.png)

If we press the search button (magnifying glass) when we are in the middle of the scroll area, the search field is not available...

- My offer is transferred the search field to a static region above the scroll zone. As it is done in the Backlinks panel.
- It would be nice to add a background for the search button when the search mode is active.

![TagFolder-01519_4](https://user-images.githubusercontent.com/125995328/220713731-e3481bce-ee1f-4369-be7e-fce94eb25ec1.png)

## 3. Fix text position in search box

Please could you correct the display of the text inside the search field

![TagFolder-01519_5](https://user-images.githubusercontent.com/125995328/220713818-4d7f002f-e8af-4d38-b77d-8dd5fc341c11.png)

## 4. Disable trim tags and filenames in sidebar

It would be very convenient if you added a switch to the Settings of TagFolder plugin

![TagFolder-01519_6](https://user-images.githubusercontent.com/125995328/220713936-98f805e1-04dc-480b-bdc9-6ecdfbd8c02b.png)

This will wrap long folder and file names to the next line.

![TagFolder-01519_7](https://user-images.githubusercontent.com/125995328/220714060-7ff341c3-0ade-4086-9b59-89d221767d71.png)

![TagFolder-01519_8](https://user-images.githubusercontent.com/125995328/220714283-16acffc1-88db-4f65-9101-9afe3f57a2ad.png)

This feature is very useful for mobile devices.

Thanks for your time and attention :)

**Comment by vrtmrz** [comment]
Thank you for asking me! No.1 to No.3 has been fixed!
Could you please try the latest version?

The wrapping behaviour of the tags can be modified by the CSS snippets as like below:
```
.tagfolder-titletagname {
    text-overflow: unset !important;
    overflow: visible !important;
    overflow-wrap: anywhere !important;
    white-space: break-spaces !important;
}
```
Would you mind if I ask you to check this as well?

**Comment by irmod** [comment]
Well done! The TagFolder pane looks marvelous! I enjoy your work!

Your CSS snippet wraps the tag folder lines perfectly. I checked.

In my case (Default Theme), I use this code:

```css
/* For Default Theme */
.tagfolder-titletagname {
    white-space: normal !important;
}
```

## Add a background for the search button

Could I ask you to add a background for the search button when the search mode is active.

![TagFolder-01520_1.png](https://user-images.githubusercontent.com/125995328/221951435-523a4a69-c9a3-4e54-94af-9d0667dbe6d8.png)

![TagFolder-01520_2.png](https://user-images.githubusercontent.com/125995328/221952078-db02238f-c53b-476d-897b-ad52839de6d5.png)

Thank you for your attention!

**Comment by vrtmrz** [comment]
Thank you for pointing it out to me! I missed this. 
It has been fixed at v0.15.21. Could you please try this version?

**Comment by irmod** [comment]
I have updated the TagFolder plugin (v0.15.21). The switch has a background when turned on.

Your powerful plugin looks marvelous! Thanks for your time
